src/components/ProductPage.jsx

```diff
@@ -0,0 +1,51 @@
+import React, {useState} from 'react';
+import ProductTable from './ProductTable';
+
+const ProductPage = () => {
+   let [inStockOnly, setInStockOnly] = useState(false);
+   let [count, setCount] = useState(0);
+
+  const handleIncrement = () => {
+    console.log('hitting increment');
+    count++;
+    setCount(count);
+  }
+
+  const handleDecrement = () => {
+    console.log('hitting decrement');
+    count--;
+    setCount(count);
+  }
+
+  // QUESITON: is initialzing handlers with a parameter the right idea? 
+  // Goal by doing this is to pass productCount down to ProductTable, 
```

Comment:
Assuming `productCount` === `count`

You wouldn't need to use the count passed from the parent to update the count because the parent already knows the count. You can just tell the parent to update the count without the arg and the parent already has all the information it need to update the count.